### PR TITLE
Refactoring code that references WinUIXamlMediaData and WinCompData to UIData.

### PR DESCRIPTION
**Pull request by plaiMS**
This is the second and final part of the refactoring. Code that will reference WinUIXamlMediaData and WinCompData is moved to UIData.

### CODE REVIEW

File: source/WinCompData/Wui/Colors.cs [hunk 1]
```diff
@@ -4,148 +4,152 @@
...
+        public static readonly Color Transparent = Color.FromArgb(0, 255, 255, 255);
+        public static readonly Color Turquoise = Color.FromArgb(255, 64, 224, 208);
+        public static readonly Color Violet = Color.FromArgb(255, 238, 130, 238);
+        public static readonly Color Wheat = Color.FromArgb(255, 245, 222, 179);
+        public static readonly Color White = Color.FromArgb(255, 255, 255, 255);
+        public static readonly Color WhiteSmoke = Color.FromArgb(255, 245, 245, 245);
+        public static readonly Color Yellow = Color.FromArgb(255, 255, 255, 0);
+        public static readonly Color YellowGreen = Color.FromArgb(255, 154, 205, 50);
     }
 }
```
Comment (sohchatt) [inline source/WinCompData/Wui/Colors.cs]: So does stylecop just complain about this (no newline at the end) but not do anything?

Comment (plaiMS) [reply source/WinCompData/Wui/Colors.cs]: Is it complaining on your machine? I don't see any complains on mine.

---
In reply to: [278657261](https://github.com/windows-toolkit/Lottie-Windows/pull/104#discussion_r278657261) [](ancestors = 278657261)

Comment (sohchatt) [reply source/WinCompData/Wui/Colors.cs]: I see it in the github ui. Just wondering how stylecop is actually enforced; not a blocker to merging.

Comment (simeoncran) [reply source/WinCompData/Wui/Colors.cs]: Stylecop runs the set of rules that we tell it to run. For some files we turn off some rules. Perhaps this was suppressed before and isn't any more.

---
In reply to: [278657261](https://github.com/windows-toolkit/Lottie-Windows/pull/104#discussion_r278657261) [](ancestors = 278657261)

File: source/WinCompData/CodeGen/CodeBuilder.cs [hunk 2]
```diff
@@ -19,17 +19,17 @@ sealed class CodeBuilder
 
         internal bool IsEmpty => _lines.Count == 0;
 
-        internal void WriteLine()
```
Comment (simeoncran) [inline source/WinCompData/CodeGen/CodeBuilder.cs]: This file should be moved into UIData and methods should go back to internal. This is really just an implementation detail of the CodeGen code.

File: source/WinCompData/Tools/Graph.cs [hunk 3]
```diff
@@ -1,120 +0,0 @@
...
-        /// <summary>
-        /// Returns the graph of nodes reachable from the given <see cref="CompositionObject"/> root.
-        /// </summary>
-        /// <returns>A <see cref="Graph"/> from the given Composition tree.</returns>
-        public static ObjectGraph<Node> FromCompositionObject(CompositionObject root, bool includeVertices)
-            => ObjectGraph<Node>.FromCompositionObject(root, includeVertices);
-
-        /// <summary>
-        /// A simple non-extensible node type.
-        /// </summary>
```
Comment (simeoncran) [inline source/WinCompData/Tools/Graph.cs]: >> [](start = 21, length = 1)

I don't see Graph and ObjectGraph in UIData. How can it build without them?

Comment (plaiMS) [reply source/WinCompData/Tools/Graph.cs]: These two files were already added to UIData in the last PR.

---
In reply to: [279119575](https://github.com/windows-toolkit/Lottie-Windows/pull/104#discussion_r279119575) [](ancestors = 279119575)

**Pull request merged**